src/utilities/effects.ts

```diff
@@ -244,6 +244,7 @@ const effects = {
         plan_id: id,
         plan_revision: revision,
         simulation_arguments: {},
+        analysis_only: false,
```

Comment:
Please alphabetize. The [pre-commit](https://github.com/NASA-AMMOS/aerie-ui/blob/develop/docs/DEVELOPER.md#before-you-commit) script should catch this for you next time.